docs/Bed_Mesh.md

```diff
@@ -410,17 +415,34 @@ adaptive_margin: 5
-Therefore, it is expected that each Gcode file will generate a mesh that probes a different
-area of the print bed. Therefore, adapted bed meshes should not be re-used. The expectation
+#### Caveats:
+
+By nature, adaptive bed meshes use the objects defined by the [exclude_object](Exclude_Object.md) module
+in the Gcode file being printed, and it is expected that each Gcode file will generate a mesh that probes a different
+area of the print bed. Therefore, adapted bed meshes should **not** be re-used. The expectation
 is that a new mesh will be generated for each print if adaptive meshing is used.
 
-It is also important to consider that adaptive bed meshing is best used on machines that can
+It is important to consider that adaptive bed meshing is best used on machines that can
 normally probe the entire bed and achieve a maximum variance less than or equal to 1 layer
 height. Machines with mechanical issues that a full bed mesh normally compensates for may
-have undesirable results when attempting print moves **outside** of the probed area. If a
-full bed mesh has a variance greater than 1 layer height, caution must be taken when using
-adaptive bed meshes and attempting print moves outside of the meshed area.
+have undesirable results when attempting print moves **outside** of the probed area. If you
+normally have a variance greater than 1 layer height when generating a full bed mesh, greater caution
+must be taken when attempting print moves outside of the adapted mesh area.
+
+If there are no objects defined by the [exclude_object](Exclude_Object.md) module before the adaptive
+bed mesh is requested, the default method will be used instead. If you are requesting an adaptive bed mesh and
+the default method is still being used, check that you have [exclude_object](Exclude_Object.md) set up correctly, and
+that the objects are being defined before the adaptive bed mesh command is executed in your Gcode file.
+
+#### How to use Adaptive Meshes:
+
+- You must have the [exclude_object](Exclude_Object.md) module enabled by adding `[exclude_object]` to your `printer.cfg` file.
+- You must enable object labeling in your slicer.
+  - Certain slicers may do this by default, or do not provide an option because it is an "always-on" feature.
+  - More modern slicers provide a "Firmware-Specific" option for labeling objects. If you enable this, you'll need to verify that your slicer supports the `klipper` gcode flavor, and is generating `EXCLUDE_OBJECT_DEFINE` commands at the beginning of the sliced gcode file.
+- If your slicer does not support firmware-specific object labeling, or does not support the `klipper` gcode flavor, you will need to [configure Moonraker's file manager module](https://moonraker.readthedocs.io/en/latest/configuration/#file_manager) to enable object processing.
+- Finally, to generate an adaptive bed mesh, you must append the `ADAPTIVE=1` flag to `BED_MESH_CALIBRATE` wherever you normally call for a bed mesh to be generated.
```

Comment:
I think a simple wording like

> Call `BED_MESH_CALIBRATE ADAPTIVE=1` either from your `[START_PRINT](Slicers.md#klipper-gcode_macro)` macro or from your slicer's starting Gcode. It is recommended to place this command pretty much at the end of the start-up routine, this means at least after heating up the bed.

would be clearer as these should be the two typical places for a regular user.

Reply:
Sure, that makes sense. I’ll find a spot to work something like this in as well.

Reply:
> I think a simple wording like
> 
> > Call `BED_MESH_CALIBRATE ADAPTIVE=1` either from your `[START_PRINT](Slicers.md#klipper-gcode_macro)` macro or from your slicer's starting Gcode. It is recommended to place this command pretty much at the end of the start-up routine, this means at least after heating up the bed.
> 
> would be clearer as these should be the two typical places for a regular user.

There is no guarantee that I user will be using a `START_PRINT`/`PRINT_START` macro. They could just be putting all of that in the slicer custom gcode.
It's better to leave it as "... add `BED_MESH_CALIBRATE ADAPTIVE=1` to your start GCode".

Reply:
Sorry, but I do not understand this comment:
![grafik](https://github.com/Klipper3d/klipper/assets/4459312/ac87809d-6fe4-4b61-8702-03d3d1884dfa)

Reply:
> Sorry, but I do not understand this comment: ![grafik](https://private-user-images.githubusercontent.com/4459312/303822480-ac87809d-6fe4-4b61-8702-03d3d1884dfa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3NTU2MDcsIm5iZiI6MTcwNzc1NTMwNywicGF0aCI6Ii80NDU5MzEyLzMwMzgyMjQ4MC1hYzg3ODA5ZC02ZmU0LTRiNjEtODcwMi0wM2QzZDE4ODRkZmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMTJUMTYyODI3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTI2NTZlYzBiMTNmZTI5OWExN2YzMDk1ZDhhYzg0OGE1NzM3ZmRhMjdhZWRiZjY2MDcyMDI0MzJkMjg2YzE1NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.FZfEpYqpDGlA5VnvJlKgibYxL9xIhV0zCskt2gLm4uw)

Oops, sorry! I didn't read the text carefully enough.